diagnostics-with-elk/azuredeploy.json

```diff
@@ -123,17 +123,12 @@
             "type": "string",
             "defaultValue": "14.04.2-LTS",
             "allowedValues": [
-                "12.04.2-LTS",
-                "12.04.3-LTS",
-                "12.04.4-LTS",
-                "12.04.5-LTS",
                 "12.10",
```

Comment:
I will add 12.04.5-LTS back as its listed on the Portal, so my allowed list will look like this:
"allowedValues": [
                "12.04.5-LTS",
                "14.04.2-LTS",
                "16.04-LTS"
            ]
Let me know if that sounds good?

BTW I did a quick search on the existing templates and there are many that reference